tsconfig.json

```
@@ -1,33 +1,19 @@
 {
   "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
-    "declaration": false,
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "lib": [
-      "dom",
-      "es2015"
-    ],
-    "module": "system",
-    "moduleResolution": "node",
+    "allowJs": true,
+    "esModuleInterop": true,
+    "jsx": "react",
+    "module": "es6",
+    "noImplicitAny": true,
```

```suggestion
   "allowJs": true,
    "declaration": true,
    "declarationMap": true,
    "esModuleInterop": true,
    "module": "es6",
    "noImplicitAny": true,
    "outDir": "dist",
```
Add declaration files. Currently rootDir = "src" is problematic, because some files require stuff outside of src folder. We will live with it for now.